src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -861,12 +858,28 @@ public VBuffer<ReadOnlyMemory<char>>[] SlotNamesMetadata(out VectorDataViewType[
     }
 
     /// <summary>
-    /// Produces a bag of counts of ngrams (sequences of consecutive words of length 1-n) in a given text.
-    /// It does so by hashing each ngram and using the hash value as the index in the bag.
-    ///
-    /// <see cref="NgramHashingEstimator"/> is different from <see cref="WordHashBagEstimator"/> in a way that <see cref="NgramHashingEstimator"/>
-    /// takes tokenized text as input while <see cref="WordHashBagEstimator"/> tokenizes text internally.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="NgramHashingTransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Vector of [Key](<xref:Microsoft.ML.Data.KeyDataViewType>) |
+    /// | Output column data type | Vector of known size of <xref:System.Single> |
+    ///
+    /// The resulting <xref:Microsoft.ML.Transforms.Text.NgramHashingTransformer/> creates a new column, named as specified in the output column name parameters, and
+    /// produces a vector of counts of n-grams (sequences of consecutive words of length 1-n) from a given data.
+    /// It does so by hashing each n-gram and using the hash value as the index in the bag.
+    ///
+    /// <xref:Microsoft.ML.Transforms.Text.NgramHashingEstimator> is different from <xref:Microsoft.ML.Transforms.Text.WordHashBagEstimator>
+    /// in a way that <xref:Microsoft.ML.Transforms.Text.NgramHashingEstimator> takes tokenized text as input while <xref:Microsoft.ML.Transforms.Text.WordHashBagEstimator> tokenizes text internally.
```

Comment:
>in a way that <xref:Microsoft.ML.Transforms.Text.NgramHashingEstimator> takes tokenized text as input while <xref:Microsoft.ML.Transforms.Text.WordHashBagEstimator> tokenizes text internally. [](start = 8, length = 191)

The former takes tokenized text as input while the latter tokenizes text internally.